cmd/juju/controller/listmodels.go

```diff
@@ -117,7 +117,7 @@ func (c *modelsCommand) Run(ctx *cmd.Context) error {
 	}
 	defer modelmanagerAPI.Close()
 
-	haveModels := false
+	var haveModels bool
```

Comment:
This is also just a matter of taste right?

Reply:
Linter has very strong tastes with, like someone who likes to drink Sambuca with every evening meal.